code/game/objects/items/teleportation.dm

```diff
@@ -219,3 +219,14 @@
 		if(active_portal_pairs[i] == P)
 			return DESTINATION_PORTAL
 	return FALSE
+
+/obj/item/hand_tele/suicide_act(mob/user)
+	if(iscarbon(user))
+		user.visible_message("<span class='suicide'>[user] is creating a weak portal and sticking their head through! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
p_their

Reply:
@ShizCalev Go back to grammar school. It looks like they're (they are) trying to commit suicide, not their trying, wtf lol.

Reply:
user.visible_message("<span class='suicide'>[user] is creating a weak portal and sticking [p_their()] head through! It looks like [user.p_theyre()] trying to commit suicide!</span>")

Reply:
But their works in that context

Reply:
yeah but pronouns